README.md

```diff
@@ -1,175 +1,302 @@
-
-        mkvirtualenv --python=[path to installed Python executable] usaspending-api
-
-6. You should see some output that looks similar to the example below. Essentially, this command creates and activates a new virtualenv named `usaspending-api` with its own set of Python libraries.  Anything you pip install from this point forward will be installed into the *usaspending-api* environment rather than your machine's global Python environment. Your command line prompt indicates which (if any) virtualenv is active.
+Decide where on your machine you want the USAspending API code to live. From the command line, navigate there and clone the USAspending API repository from GitHub to your local environment:
 
- **Note:** in the command below, replace `/usr/local/bin/python3.5` with the path to your local Python executable.
+```
+$ git clone https://github.com/fedspendingtransparency/usaspending-api.git
+```
 
-        $ mkvirtualenv --python=/usr/local/bin/python3.5 usaspending-api
-        Running virtualenv with interpreter /usr/local/bin/python3.5
-        New python executable in usaspending-api/bin/python3.5
-        Also creating executable in usaspending-api/bin/python
-        Installing setuptools, pip...done.
 
-        (usaspending-api)$
+Now we will allow `venv` to manage the virtual environment by the following:
+```
+$ echo "layout python-venv python3.5" >> usaspending-api/.envrc
+$ cd usaspending-api
+$ direnv allow
+$ source activate
+```
 
-7. This new environment will be active until you run the `deactivate` command. You can re-activate the environment again at any time by typing `workon usaspending-api`.
+The `source activate` command will open the virtual environment for usaspending-api. Anything you pip install from this point forward will be installed into the *usaspending-api* environment rather than your machine's global Python environment. Your command line prompt indicates which (if any) virtual environment is active.
 
-### USAspending API Backend
+  
 
-Decide where on your machine you want the USAspending API code to live. From the command line, navigate there and clone the USAspending API repository from GitHub to your local environment:
+*Remember to use `source activate` as you work in the project folder; You can exit the virtual environment by using `deactivate`*
 
-        $ git clone https://github.com/fedspendingtransparency/usaspending-api.git
+  
 
-Navigate to the USAspending API's main folder:
+## Install the dependencies
+Use `source activate` to open the virtual environment
+```
+$ source activate
+(python-venv-3.5.0) $ pip install -r requirements/requirements.txt
+```
 
-        $ cd usaspending-api
+  
 
+Install caching dependencies. (This is optional, and should only be done if you have installed Memcached and require AWS caching for development)
 
-Install the dependencies.
+```
+$ pip install -r requirements/caching_requirements.txt
```

Comment:
include "(python-venv-3.5.0)" or remove from previous pip install instruction for consistency